heron/api/src/java/shade.conf

```diff
@@ -0,0 +1 @@
+rule com.google** com.twitter.heron.shaded.@0
```

Comment:
I recommend to make name worse (with __shaded__ for example if possible) to make it impossible to pass the review in user code with accidental usages.